source/frontend/config-overrides.js

```diff
@@ -14,11 +14,19 @@ const addCompressionPlugin = config => {
   return config;
 };
 
-module.exports = function override(config) {
-  const fallback = config.resolve.fallback || {};
+module.exports = {
+  // The Webpack config to use when compiling your react app for development or production.
+  webpack: function override(config) {
+    const fallback = config.resolve.fallback || {};
 
-  config.resolve.fallback = fallback;
-  addCompressionPlugin(config);
-  config.ignoreWarnings = [/Failed to parse source map/, /autoprefixer/];
-  return config;
+    config.resolve.fallback = fallback;
+    addCompressionPlugin(config);
+    config.ignoreWarnings = [/Failed to parse source map/, /autoprefixer/];
+    return config;
+  },
+  // The Jest config to use when running your jest tests
+  jest: function override(config) {
+    config.transformIgnorePatterns = ['node_modules/(?!(react-leaflet|@react-leaflet/core)/)'];
+    return config;
+  },
```

Comment:
This change is required to make jest aware that the new react-leaflet library has no CommonJS exports (like most libraries out there)

Not sure why react-leaflet would choose to only release in the newer ES module (ESM) format that is not as widely supported by existing tooling. But we don't control how the library author publishes his library